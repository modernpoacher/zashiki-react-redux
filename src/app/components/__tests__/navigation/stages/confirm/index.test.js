import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16'

import Confirm from '@modernpoacher/zashiki-react-redux/app/components/navigation/stages/confirm'

import getConfirmRoute from '@modernpoacher/zashiki-react-redux/app/common/get-confirm-route'

Enzyme.configure({ adapter: new Adapter() })

jest.mock('@modernpoacher/zashiki-react-redux/app/common/get-confirm-route')

jest.mock('react-router-dom')

describe('@modernpoacher/zashiki-react-redux/app/components/navigation/stages/confirm', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Confirm />)
  })

  it('renders', () => {
    expect(toJson(wrapper))
      .toMatchSnapshot()
  })

  it('invokes `getConfirmRoute`', () => {
    expect(getConfirmRoute)
      .toBeCalled()
  })
})
