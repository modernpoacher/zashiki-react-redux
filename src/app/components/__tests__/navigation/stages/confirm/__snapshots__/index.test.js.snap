// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@modernpoacher/zashiki-react-redux/app/components/navigation/stages/confirm renders 1`] = `ShallowWrapper {}`;
